lumigator/python/mzai/backend/jobs/submission.py

```diff
@@ -25,8 +25,11 @@ class RayJobEntrypoint(ABC):
 
     @property
     def command(self) -> str:
-        """The command ran by the ray job, which is passed a JSON-serialized config."""
-        return f"lm-buddy evaluate huggingface --config '{json.dumps(self.config.args)}'"
+        # lm-buddy passed as a module to Ray using a JSON-serialized config.
+        return (
+            f"python -m lumigator.python.mzai.backend.lm_buddy evaluate huggingface "
```

Comment:
It's based on Ray's architecture. We pass this command,

```lm-buddy evaluate huggingface --config '{json.dumps(self.config.args)}****```,

Because [Ray entrypoints require](https://github.com/ray-project/ray/blob/a1be06346e532e656c128ffd2230f06a4d72679e/python/ray/dashboard/modules/job/sdk.py#L149) a CLI for jobs, whether that's a Python script or a bash script, or some combination.  We also need to pass configuration into this CLI call as args.